_i18n/en/resources/developer-guides/daemon-rpc.md

```diff
@@ -563,270 +783,1636 @@ $ curl -X POST http://127.0.0.1:18081/json_rpc -d '{"jsonrpc":"2.0","id":"0","me
+
+{
+  "pool_stats": {
+    "bytes_max": 47222,
+    "bytes_med": 13290,
+    "bytes_min": 13092,
+    "bytes_total": 449511,
+    "fee_total": 289715320000,
+    "histo": "\t▒'▒5▒4▒\/▒▒▒$3",
+    "histo_98pc": 0,
+    "num_10m": 18,
+    "num_double_spends": 1,
+    "num_failing": 17,
+    "num_not_relayed": 0,
+    "oldest": 1525457001,
+    "txs_total": 26
+  },
+  "status": "OK",
+  "untrusted": false
+}
+```
+
+
+### **/stop_daemon**
+
+Send a command to the daemon to safely disconnect and shut down.
+
+Alias: *None*.
+
+Inputs: *None*.
+
+Outputs:
+
+* *status* - string; General RPC error code. "OK" means everything looks good.
+
+Example:
+
+```
+$ curl -X POST http://127.0.0.1:18081/stop_daemon -H 'Content-Type: application/json'
+
+{
+  "status": "OK"
+}
+```
+
+
+### **/get_info (legacy)**
+
+This method is a convenient backward support and should not be used anymore. See [get_info](#get_info) JSON RPC for details.
+
+Alias:
+
+* */getinfo*
+* *get_info*
+
+
+### **/get_limit**
+
+Get daemon bandwidth limits.
+
+Alias: *None*.
+
+Inputs: *None*.
+
+Outputs:
+
+* *limit_down* - unsigned int; Download limit in kBytes
+* *limit_up* - unsigned int; Upload limit in kBytes
+* *status* - string; General RPC error code. "OK" means everything looks good.
+* *untrusted* - boolean;
+
+Example:
+
+```
+$ curl -X POST http://127.0.0.1:18081/get_limit -H 'Content-Type: application/json'
+
+{
+  "limit_down": 8192,
+  "limit_up": 128,
+  "status": "OK",
+  "untrusted": false
+}
+```
+
+
+### **/set_limit**
+
+Set daemon bandwidth limits.
+
+Alias:
+
+* *limit_down* - unsigned int; Optional, download limit in kBytes
+* *limit_up* - unsigned int; Optional, upload limit in kBytes
+
+Inputs: *None*.
+
+Outputs:
+
+* *limit_down* - unsigned int; Download limit in kBytes
+* *limit_up* - unsigned int; Upload limit in kBytes
+* *status* - string; General RPC error code. "OK" means everything looks good.
+
+Example:
+
+```
+$ curl -X POST http://127.0.0.1:18081/set_limit -d '{"limit_down": 1024}' -H 'Content-Type: application/json'
+
+{
+  "limit_down": 1024,
+  "limit_up": 128,
+  "status": "OK"
+}
+```
+
+
+### **/out_peers**
+
+Limit number of Outgoing peers.
+
+Alias: *None*.
+
+Inputs:
+
+* *out_peers* - unsigned int; Max number of outgoing peers
+
+Outputs:
+
+* *status* - string; General RPC error code. "OK" means everything looks good.
+
+Example:
+
+```
+$ curl -X POST http://127.0.0.1:18081/out_peers -d '{"out_peers": 3232235535}' -H 'Content-Type: application/json'
+
+{
+  "status": "OK"
+}
+```
+
+
+### **/in_peers**
+
+Limit number of Incoming peers.
+
+Alias: *None*.
+
+Inputs:
+
+* *in_peers* - unsigned int; Max number of incoming peers
+
+Outputs:
+
+* *status* - string; General RPC error code. "OK" means everything looks good.
+
+Example:
+
+```
+$ curl -X POST http://127.0.0.1:18081/out_peers -d '{"in_peers": 3232235535}' -H 'Content-Type: application/json'
+
+{
+  "status": "OK"
+}
+```
+
+
+### **/start_save_graph**
+
+Start saving graph.
```

Comment:
What is that graph ?

Reply:
Thanks for asking. I cannot figure it out.